### PR TITLE
Update anonaddy extension

### PR DESCRIPTION
**Pull request by mormubis**
## Description
AnonAddy has been rebranded to Addy. This PR updates the company name and API endpoints to their new site.

Additionally, new icons have been added to it.

This PR refactors the old `anonaddy` extension and fixes the issue with the default domain when creating an alias.

## Screencast

Screenshot 2023-12-03 at 4 16 56 PM

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/anonaddy/package.json [hunk 1]
```diff
@@ -1,67 +1,67 @@
...
+    {
+      "name": "edit-alias",
+      "title": "Edit Alias",
+      "subtitle": "Addy",
+      "description": "Edit an Addy alias",
+      "mode": "view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.39.3",
```
Comment (pernielsentikaer) [inline extensions/anonaddy/package.json:40]: Should we bump this?

File: extensions/anonaddy/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
-# AnonAddy Changelog
+# Addy Changelog
+
+## [Unreleased] - 2023-12-03
```
Comment (pernielsentikaer) [inline extensions/anonaddy/CHANGELOG.md:3]: You can use write [Enhancements]

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @mormubis 👋 

Thanks for your contribution 🔥

@httpjamesm do you mind checking this?

**Comment by httpjamesm** [comment]
> Hi @mormubis 👋 
> 
> Thanks for your contribution 🔥
> 
> @httpjamesm do you mind checking this?

Looks ok to me

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**